src/main.js

```diff
@@ -38,7 +38,7 @@ app.extend({
   }
 });
 
-app.initialize();
-
 // export app to console
 window.app = app;
+
+app.initialize();
```

Comment:
Any reason for the change in order? Is this so that `window.app` is available to the views below? If so, I'd actually suggest we move `app.initialize` back to its original location, to prevent leaking `window` refs throughout the code :-)

Not expecting `window` to exist will, someday, make unit testing a lot easier